pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -146,13 +149,26 @@ stdenv.mkDerivation (finalAttrs: ({
     patchShebangs ./bin
   '' + (
     if stdenv.hostPlatform.isDarwin then ''
+      for file in \
+        configure \
+        mkspecs/features/mac/asset_catalogs.prf \
+        mkspecs/features/mac/default_pre.prf \
+        mkspecs/features/mac/sdk.mk \
+        mkspecs/features/mac/sdk.prf
+      do
+        substituteInPlace "$file" \
+          --replace-quiet '/usr/bin/xcode-select' '${lib.getBin xcbuild}/bin/xcode-select' \
+          --replace-quiet '/usr/bin/xcrun' '${lib.getBin xcbuild}/bin/xcrun' \
+          --replace-quiet '/usr/libexec/PlistBuddy' '${lib.getBin xcbuild}/bin/PlistBuddy'
```

Comment:
Would it be better to use `lib.getExe'`?

Reply:
Sure, why not. (Pushed some miscellaneous other fixes and building again.)